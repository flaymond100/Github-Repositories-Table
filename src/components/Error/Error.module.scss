.errorWrapper {
  display: flex;
  justify-content: center;
  margin-top: 20%;
}
