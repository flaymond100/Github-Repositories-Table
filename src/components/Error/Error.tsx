import { FC } from "react";
import { Alert, AlertTitle } from "@material-ui/lab";
import { useTranslation } from "react-i18next";
import styles from "./Error.module.scss";

const Error: FC = () => {
  const [translation] = useTranslation();

  return (
    <div className={styles.errorWrapper}>
      <Alert severity="error">
        <AlertTitle>{translation("error.title")}</AlertTitle>
        {translation("error.text")}
      </Alert>
    </div>
  );
};

export default Error;
