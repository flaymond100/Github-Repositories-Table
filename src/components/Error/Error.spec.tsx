import * as React from "react";
import { shallow } from "enzyme";
import Error from "./Error";

describe("render Error component", () => {
  test("should render Error component", () => {
    const component = shallow(<Error />);

    expect(component).toMatchSnapshot();
  });
});
