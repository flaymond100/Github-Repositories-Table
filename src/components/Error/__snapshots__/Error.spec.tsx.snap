// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Error component should render Error component 1`] = `
<div
  className="errorWrapper"
>
  <WithStyles(ForwardRef(Alert))
    severity="error"
  >
    <WithStyles(ForwardRef(AlertTitle))>
      error.title
    </WithStyles(ForwardRef(AlertTitle))>
    error.text
  </WithStyles(ForwardRef(Alert))>
</div>
`;
